.github/workflows/ansible-test-plugins.yml

```diff
@@ -27,20 +27,17 @@ jobs:
           - stable-2.9
           - stable-2.10
           - devel
-        python:
-          - 2.7
-          - 3.8
     steps:
 
       - name: Check out code
         uses: actions/checkout@v2
         with:
           path: ansible_collections/community/proxysql
 
-      - name: Set up Python ${{ matrix.python }}
+      - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: ${{ matrix.python }}
```

Comment:
Can't GitHub CI be configured so that it only checks using python version 3.8, therefore you don't have to increment python version once in a while?

Reply:
I'm not sure what you mean. This change sets the version to 3.8.

Reply:
I suppose `${{ matrix.python }}` has two values based on matrix.python which has been set above, once 2.7, once 3.8. You think it's unnecessary and are willing to change it to 3.8 in order to prevent creating two tests.

If you don't delete the matrix.python (lines 30-32) and just remove 2.7 (line 31), you don't need to change it all over the yaml file, you can just change it once.

But I was thinking couldn't it be set python:latest for example or something like that, so that you don't need to increment python version every year (or whenever python 3.9 is out).

Reply:
In the whole sanity test part of the CI description, the Python version is used precisely once, so it doesn't matter whether it is mentioned in the matrix or in the setup-python step. Also, it only needs to be *a* version supported by Ansible, so 3.8 can probably stay there for a very long time (until ansible-base drops support for Python 3.8).

Setting it to latest is not a good idea, since that could result in random breakages when ansible-base happens to not support a Python version.

Reply:
You're right, thanks for explaining!